mzLib/Proteomics/Proteomics.csproj

```diff
@@ -9,6 +9,11 @@
     <DebugType>full</DebugType>
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="libtorch-cpu" Version="2.1.0.1" />
```

Comment:
concerned about support for things we need. i'm confused on this matter.
![image](https://github.com/smith-chem-wisc/mzLib/assets/16841846/729a4170-7772-4909-a1d7-2b120e02060a)

Reply:
This is the LibTorch backend package. It's what the TorchSharp nuget uses. It a "wrapper library" for LibTorch

https://github.com/dotnet/TorchSharp?tab=readme-ov-file#torchsharp